shared/common-adapters/name-with-icon/index.tsx

```diff
@@ -290,7 +290,7 @@ const styles = Styles.styleSheetCreate(() => ({
     },
   }),
   textBold: {
-    fontWeight: 700,
+    fontWeight: '700',
```

Comment:
The integer doesn't work? Have you tested this on mobile?

Reply:
yeah it doesn't work on mobile.